docs/serving/services/configure-probing.md

```diff
@@ -0,0 +1,81 @@
+# Configuring Probing
+
+## General understanding of Knative Probing
+
+It is important to note that Knative probing is different from Kubernetes probing. 
+One reason for this is that Knative tries to minimize cold-start time and thus is probing
+in a vastly higher interval than Kubernetes does. 
+
+The general probing architecture looks like this:
+
+![probing-overview](./probes-overview.drawio.svg)
+
+* Users can optionally define Readiness and/or Liveness probes in the `KnativeService` CR.
+* The Liveness probes are directly executed by the Kubelet against the according container.
+* Readiness probes, on the other hand, are rewritten by Knative to be executed by the Queue-Proxy container to allow more aggressive probing and faster cold-start times.
```

Comment:
Could you explain a bit more the aggressive probing and who does probing eg. activator, ingress (at least as abackground info this info valuable for regular users in order to understand a bit of the internals and during debugging).